### PR TITLE
Alxngyn/users migration

### PR DESCRIPTION
**Pull request by alxngyn**
fixes issue #122
Blocked until #144 is merged

## Changes in this PR.

- [X] Deleted blank lines/guest accounts in the users csv file
- [X] Created `get_users.py` to import the data into the database
- [X] Handle the sensitive data

## Testing this PR.

1. Do `docker-compose run web bash`.
2. Do `cd data_scripts`.
3. Do `python get_schools.py`
4. Do `python get_users.py`
5. `cd ..`
6. Do `python streamwebs_frontend/manage.py dbshell`
7. `SELECT * FROM auth_user;`  gives username/pw/email
8. `SELECT * FROM streamwebs_userprofile;` gives birthday & school
9. OPTIONAL: `SELECT COUNT(*) FROM auth_user;` should give you an integer count and should match `SELECT COUNT(*) FROM streamwebs_userprofile;`

### Expected Output.

```
streamwebs=# select count(*) from auth_user;
 count 
-------
   6
(1 row)

streamwebs=# select count(*) from streamwebs_userprofile;
 count 
-------
   6
(1 row)
```

@osuosl/devs

### CODE REVIEW

File: csvs/users2.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Uid,First name,Last name,Date of birth,School
+1,Alex,Win,"March 1, 1978",Muddy Charter School
```
Comment (leian7) [inline csvs/users2.csv]: For userprofile, it's not matching for me.
```
streamwebs=# select * from streamwebs_userprofile;
 id |                school                 | birthdate  | user_id 
----+---------------------------------------+------------+---------
  1 | []                                    | 1978-03-01 |       1
  2 | [<School: David Douglas High School>] | 1928-01-19 |       2
  3 | []                                    | 1999-08-26 |       3
  4 | [<School: Health & Science School>]   | 1999-09-04 |       4
  5 | [<School: Health & Science School>]   | 2001-05-30 |       5
  6 | [<School: Sunset High School>]        | 1970-01-01 |       6
(6 rows)
```

**Comment by leian7** [comment]
+1

File: data_scripts/get_users.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                        dob = datetime.strptime(row2["Date of birth"], '%B %d, %Y')
+                    else:
+                        dob = "1970-1-1"
+
+                    if row2["School"]:
+                        users_school = row2["School"]
+                    else:
+                        school = ""
+
+        user = User.objects.create_user(username=username,
```
Comment (athai) [inline data_scripts/get_users.py]: Why are you creating a user, school, and userprofile here?

Comment (alxngyn) [inline data_scripts/get_users.py]: The user and userprofile objects write to the database. The school object is to find the school that matches the name given from the csv.

Comment (athai) [inline data_scripts/get_users.py]: Oh, okay. I see what you're doing now. +1

**Pull request merged**